tone-analyzer/v3.js

```diff
@@ -83,4 +83,26 @@ ToneAnalyzerV3.prototype.tone = function(params, callback) {
   return requestFactory(parameters, callback);
 };
 
+ToneAnalyzerV3.prototype.tone_chat = function(params, callback) {
+  if (!params || !params.utterances) {
+    callback(new Error('Missing required parameters: utterances'));
```

Comment:
Updated: https://github.com/aprilwebster/node-sdk/blob/master/tone-analyzer/v3.js#L93

Should I take care of the params check for the tone endpoint too?  https://github.com/aprilwebster/node-sdk/blob/master/tone-analyzer/v3.js#L68